index.html

```diff
@@ -120,13 +125,39 @@ <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
             See Project
           </a>
         </article>
+        <article class="work">
+          <img src="./media/printing.png" alt="first project screenshot" class="work-image">
+          <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="language-tags">
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">JavaScript</li>
+            <li class="tag">CSS</li>
+            <li class="tag">HTML</li>
+          </ul>
+          <a href="#" class="work-link">
+            See Project
+          </a>
+        </article>
+        <article class="work">
+          <img src="./media/printing.png" alt="first project screenshot" class="work-image">
+          <h3 class="work-title">Multi-Post Stories Gain+Glory</h3>
+          <ul class="language-tags">
+            <li class="tag">Ruby on rails</li>
+            <li class="tag">JavaScript</li>
+            <li class="tag">CSS</li>
+            <li class="tag">HTML</li>
```

Comment:
- [ ] Good job so far 🌜, as [requested](https://github.com/Ellon-M/portfolio/pull/7#discussion_r1014536405) by the previous reviewer, kindly, your design should sticks as much as possible with Figma design, please, fix it.

Your design:

![image](https://user-images.githubusercontent.com/91301423/200096189-8871985b-d857-4791-87c7-ebc0183fea2b.png)

Figma design:

![image](https://user-images.githubusercontent.com/91301423/200096230-510a1fee-8dac-4f6f-81a5-c6bc249990b1.png)